pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -58,7 +58,9 @@ func (alloc *Action) Execute(ssn *framework.Session) {
 	jobsMap := map[api.NamespaceName]map[api.QueueID]*util.PriorityQueue{}
 
 	for _, job := range ssn.Jobs {
-		if job.PodGroup.Status.Phase == scheduling.PodGroupPending {
+		if job.PodGroup.Status.Phase == scheduling.PodGroupPending || job.PodGroup.Status.Phase == "" {
```

Comment:
l believe there are more code having this issue, pls help check it out.
and it's better to abstract a func to do that

Reply:
OK, define the function "jobinfo.IsPending() bool" to check it.